api/api_ethereum.go

```diff
@@ -308,7 +311,8 @@ func (api *EthereumAPI) GasPrice(ctx context.Context) (*hexutil.Big, error) {
 
 // MaxPriorityFeePerGas returns a suggestion for a gas tip cap for dynamic fee transactions.
 func (api *EthereumAPI) MaxPriorityFeePerGas(ctx context.Context) (*hexutil.Big, error) {
-	return api.publicKlayAPI.MaxPriorityFeePerGas(ctx)
+	// TODO-Klaytn maybe need to branch kip71 hardfork
```

Comment:
We'd better make sure this. Do we need to do different action based on the hardfork or not?

Reply:
I think we can remove this `TODO` comment.
`api.GasPrice` will return unit price before hard fork and return next block'sm baseFee after hard fork.
@bobpkr Please let me know if i misunderstood

Reply:
We need to discuss this! What to do with maxPriorityFeePerGas...! Please share your opinions.

Reply:
First of all, you are right.
I heard that @kjeom will be more organized and will give you some feedback.